ci/sawtooth-build-docs

```diff
@@ -13,6 +13,19 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# Description:
+#   Builds the environment needed to build the Sawtooth Lake docs
+#   Running the image will put the Sawtooth Lake docs in
+#   sawtooth-core/docs/build
```

Comment:
Amend this to read "Running the image will put the Sawtooth Lake docd in sawtooth-core/docs/build on your local machine."